kinto/__main__.py

```diff
@@ -143,6 +153,11 @@ def main(args=None):
                                          parsed_args['bucket'],
                                          parsed_args['collection'])
 
+    elif which_command == 'rebuild-quotas':
+        dry_run = parsed_args['dry_run']
+        env = bootstrap(config_file)
+        return scripts.rebuild_quotas(env, dry_run=dry_run)
```

Comment:
I decided to keep the wrapper (that checks config parameters and returns exit status codes) in scripts, but move the body of the script into the quota plugin.